release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -7,11 +7,37 @@
 
 A number of security fixes have been applied in this release, including the following:
 
-* NodeJS update for RHSA 128590, see https://errata.devel.redhat.com/advisory/128590
+* link:https://errata.devel.redhat.com/advisory/128590[NodeJS update for RHSA 128590] 
 * link:https://access.redhat.com/security/cve/CVE-2024-28176[CVE-2024-28176] Jose was found to have an uncontrolled resource consumption vulnerability (ocm-backend plugin).
 * link:https://access.redhat.com/security/cve/CVE-2024-28849[CVE-2024-28849] A vulnerability was found in the follow-redirects package. Follow redirects updated to 1.15.6.
 
-== Enhancements and bug fixes in {product} 1.1.0 and 1.1.1
+== Enhancements and bug fixes in {product} 1.1.0, 1.1.1, and 1.1.2
```

Comment:
There will be a 1.1.3; do you really want to keep adding to this list? 
```suggestion
== Enhancements and bug fixes in {product} 1.1.z
```

Reply:
IBM style guidance on [References to multiple versions](https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#references-to-multiple-releases) is to list al the versions the way that @Gerry-Forde has done and to "not use a convention such as 5.x or 5.n"

Reply:
Thanks for confirming that @linfraze